english/simpsons-joke/joke.md

```diff
@@ -0,0 +1,7 @@
+KEEP OUT
+
+Or Enter,
+I'm a sign
+NOT A COP
+
+Click [here](https://github.com/gautamhans1/create-your-own-adventure/commit/437c51818bed6c0778985c8823d6ba3e7c20a7f4)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.